apps/dsp-app/src/config/config.dev-server.json

```
@@ -31,7 +31,8 @@
       "console": {
         "enabled": true,
         "disabledLevels": []
-      }
+      },
+      "tracingCorsUrls": []
```

```suggestion
      "tracingCorsUrls": ["https://api.dev.dasch.swiss", "https://ingest.dev.dasch.swiss"]
```
Wouldn't it make sense to add these here?